kafka/conn.py

```diff
@@ -220,6 +221,15 @@ def _wrap_ssl(self):
                 self._ssl_context.load_cert_chain(
                     certfile=self.config['ssl_certfile'],
                     keyfile=self.config['ssl_keyfile'])
+            if self.config['ssl_crlfile']:
+                if not hasattr(ssl, 'VERIFY_CRL_CHECK_LEAF'):
+                    log.error('No CRL support with this version of Python.'
+                              ' Disconnecting.', self)
```

Comment:
Yes, you are right. However, this code path shouldn't have hit Travis. Let me investigate a bit. I keep `self` since it should be the same as `str(self)` and most of the code in the same file uses `self`.